src/CallInvite.tsx

```diff
@@ -183,6 +184,34 @@ export class CallInvite {
   getTo(): string {
     return this._to;
   }
+
+  /**
+   * Update the caller name displayed in the iOS system incoming call screen.
+   *
+   * @param newHandle - The new value of the caller's name.
+   *
+   * @remarks
+   * Unsupported platforms:
+   * - Android
+   *
+   * This API is specific to iOS and unavailable in Android.
+   *
+   * @returns
+   *  - Resolves when the caller name has been updated.
+   */
+  updateCallerHandle(newHandle: string): Promise<void> {
```

Comment:
We should make this an `async` function, that way when we throw, it will get wrapped into a Promise that rejects.